pr-to-review/pr-to-review.2m.sh

```diff
@@ -3,31 +3,26 @@
 ######### CONFIGURATION #########
 GITHUB_USER="" # e.g. "gabro"
 GITHUB_COM_TOKEN="" # get one at https://github.com/settings/tokens
-GITHUB_ENTERPRISE_TOKEN="" # get one at https://github.omnilab.our.buildo.io/settings/tokens
 #################################
 
 PATH=$PATH:/usr/local/bin
 
 GITHUB_COM_API="https://api.github.com"
-GITHUB_ENTERPRISE_API="https://github.omnilab.our.buildo.io/api/v3"
 
 QUERY="review-requested:$GITHUB_USER+type:pr+state:open"
 
 GITHUB_COM_RESULT=`curl -ks $GITHUB_COM_API/search/issues\?q\=$QUERY\&access_token\=$GITHUB_COM_TOKEN`
-GITHUB_ENTERPRISE_RESULT=`curl -ks $GITHUB_ENTERPRISE_API/search/issues\?q\=$QUERY\&access_token\=$GITHUB_ENTERPRISE_TOKEN`
 
 GITHUB_COM_COUNT=`echo $GITHUB_COM_RESULT | jq '.total_count'`
-GITHUB_ENTERPRISE_COUNT=`echo $GITHUB_ENTERPRISE_RESULT | jq '.total_count'`
 
-PR_COUNT=$(($GITHUB_COM_COUNT + $GITHUB_ENTERPRISE_COUNT))
+PR_COUNT=$(($GITHUB_COM_COUNT))
```

Comment:
```suggestion
PR_COUNT=$GITHUB_COM_COUNT
```

Volendo potremmo levare la variabile del tutto ma cambia poco